test/InterchainTokenService.js

```diff
@@ -621,6 +621,8 @@ describe('Interchain Token Service', () => {
                     'Call deployInterchainToken on source chain',
                 ),
             )
+                .to.emit(service, 'InterchainTokenIdClaimed')
```

Comment:
also add for the case when deploying on remote chain, and not the current one

Reply:
that's good call, updated.